templates/includes/records/hierarchy-global.html

```diff
@@ -1,29 +1,236 @@
 {% load records_tags %}
 
-<nav class="hierarchy-global" data-container-name="hierarchy-global" id="analytics-hierarchy-global">
-
-    <details id="js-hierarchy-global" open>
-        <summary id="analytics-hierarchy-link">
-            <h2 class="hierarchy-global__heading" id="hierarchy-label">{{ 'hierarchy'|as_label }}</h2>
-        </summary>
-
-        <ol class="hierarchy-global__list" aria-labelledby="hierarchy-label">
-            {% for item in page.hierarchy %}
-                {% is_page_current_item_in_hierarchy page item as is_current_item %}
-                {% if is_current_item %}
-                    <li class="hierarchy-global__list-item hierarchy-global__list-item--current-item">
-                        <span class="hierarchy-global__location">You are here:</span>
-                        <span class="hierarchy-global__reference hierarchy-global__reference--white">{{ item.reference_number }}</span> {{ item.title }}
-                    </li>
+<div class="hierarchy" data-container-name="hierarchy-global">
+    <div class="hierarchy__container">
+
+        <!-- SHORT HIERARCHY VIEW -->
+        <nav class="hierarchy-short-panel" role="navigation">
+            <h2 class="hierarchy-short-panel__heading sr-only">Catalogue location at a glance</h2>
+            <p class="hierarchy-short-panel__label">You are here:</p>
+
+            <ul class="hierarchy-short-panel__list">
+                {% if page.hierarchy %}
+                    {% if page.hierarchy|length <= 2 %}
+                        <li class="hierarchy-short-panel__list-item">
+                            <span class="hierarchy-short-panel__level">
+                                <a href="#">Archive</a>
+                            </span>({{ page.template.heldBy }})
+                        </li>
+                    {% endif %}
+                    
+                    {% for item in page.hierarchy %}
+                        {% is_page_current_item_in_hierarchy page item as is_current_item %}
+
+                        {% if forloop.counter == 1 and not is_current_item %}
+                            {% if page.hierarchy|length == 3 %}
+                                <li class="hierarchy-short-panel__list-item hierarchy-short-panel__list-item--more">
+                                    <span class="hierarchy-short-panel__level">
+                                        <a href="{% record_url item%}">Department</a>
+                                    </span>({{ item.reference_number }})
+                                </li>
+                            {% elif page.hierarchy|length == 2 %}
+                                <li class="hierarchy-short-panel__list-item">
+                                    <span class="hierarchy-short-panel__level">
+                                        <a href="{% record_url item%}">Department</a>
+                                    </span>({{ item.reference_number }})
+                                </li>
```

Comment:
```
<span class="hierarchy-short-panel__level">
    <a href="{% record_url item%}">Department</a>
</span>({{ item.reference_number }})
```

- Some bits is repetitive, could we do an include tag here 
- `Department` - is hardcoded - perhaps we could derive it through a template tag for that level.
- Similar for `Series`, `Piece`